src/tasks/etape-update/titre-etape-communes-update.js

```diff
@@ -7,7 +7,7 @@ import { geojsonFeatureMultiPolygon } from '../../tools/geojson'
 import geojsonCommunesGet from '../../tools/api-communes'
 
```

Comment:
to-do: le log qui est retourné à la fin de la fonction est incomplet: il manque le nombre de communes ajoutées dans la bdd. On peut s'inspirer de ce qui est fait dans `/tasks/daily/titres-etapes-communes-update` et retourner un tableau avec 2 lignes de log. qu'en penses tu ?

Reply:
Bonne idée. C'est fait. 👍